src/index.ts

```diff
@@ -15,12 +15,13 @@
  */
 `use strict`;
```

Comment:
Unrelated to this patch, but we should fix it: this line has no effect. It should be `'use strict'` or `"use strict"`.

Compare e.g.

```
(function() { 'use strict'; return this; })();
// → undefined, i.e. strict mode is enabled

(function() { `use strict`; return this; })();
// → globalThis, strict mode is not enabled
```

Reply:
I wasn't aware of that difference. Thanks!